content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -1,4 +1,4 @@
 ---
 layout: redirect
-redirect_url: /doc/book/platform-information/upgrade-java-to-11/
+redirect_url: /doc/book/platform-information/upgrade-java-to-17/
```

Comment:
I think that the redirect URL needs to continue to point to the Java 11 upgrade guide because that was the content of the file at the previous location.  The redirect in this case is assuring that links to the Java 11 upgrade guide will remain links to the Java 11 upgrade guide.
```suggestion
redirect_url: /doc/book/platform-information/upgrade-java-to-11/
```